src/common/user-store.ts

```diff
@@ -97,6 +113,12 @@ export class UserStore extends BaseStore<UserStoreModel> {
    */
   hiddenTableColumns = observable.map<string, ObservableToggleSet<string>>();
 
+  editorConfiguration = observable.map<EditorType, EditorConfiguration>([
+    [EditorType.TABS, {miniMap: {enabled: true}, lineNumbers: "on", tabSize: 4}],
+    [EditorType.DETAILS, {miniMap: {enabled: false}, lineNumbers: "on", tabSize: 2}],
+    [EditorType.ADD_CLUSTER, {miniMap: {enabled: false}, lineNumbers: "on", tabSize: 2}],
+    [EditorType.KUBECONFIG, {miniMap: {enabled: false}, lineNumbers: "off", tabSize: 2}]]);
```

Comment:
Please rebase/merge master as this sort of thing has now changed.

Furthermore I think it would be better if we just had settings for all kinds of editors to begin with, this fine detail seems a bit much.

Reply:
I'm not sure that is necessary. For instance, all tabs have the same position, size, and similar data, I believe that nobody will be in need to have different configs for creating and editing pods(deployments/etc).

Reply:
Not sure, first - default settings (ace editor) are different, some places have no line-numbers
second - some places are too small to show minimap for navigation (imho)
therefore I would like to have the ability to configure each mount point of the editor.

Reply:
How about a narrow/wide approach like https://github.com/lensapp/lens/pull/2949#discussion_r668750681?